_articles/upgrade-pop.md

```diff
@@ -21,53 +22,60 @@ section: pop-ubuntu
 
 ---
 
-Pop!_OS 18.10 was released in October, 2018. This article explains how to upgrade from Pop!_OS 18.04 to Pop!_OS 18.10.
+### Upgrading to Pop!\_OS 19.10
 
-Pop!_OS 19.04 was released in April, 2019. This article explains how to upgrade from Pop!_OS 18.10 to Pop!_OS 19.04.
+> NOTE: Users running Pop!_OS 18.04 LTS may also upgrade to Pop!_OS 19.10 (if they wish)
```

Comment:
This isn't going to be available until sometime November.

Reply:
Removed this line, and we can add it back once that feature is available. (The article already mentions that it will be possible in November later.)